### PR TITLE
update Listservices filter

### PR DESCRIPTION
**Pull request by fengxx**
add missing label filter for docker service

---
This change is [Reviewable](https://reviewable.io/reviews/docker-java/docker-java/1002)

### CODE REVIEW

File: src/main/java/com/github/dockerjava/api/command/ListServicesCmd.java [hunk 1]
```diff
@@ -18,17 +18,36 @@
...
-     *            - Show only services with the given ids
+     * @param ids - Show only services with the given ids
      */
     ListServicesCmd withIdFilter(List<String> ids);
+
+    /**
+     * @param names - Show only services with the given names
+     */
+
+    ListServicesCmd withNameFilter(String... names);
```
Comment (KostyaSha) [inline src/main/java/com/github/dockerjava/api/command/ListServicesCmd.java]: method with list already exist, why do you nead additional?

Comment (fengxx) [reply src/main/java/com/github/dockerjava/api/command/ListServicesCmd.java]: existing signature is List, add Varargs to get consistent with existing ListTasksCmd

Comment (KostyaSha) [reply src/main/java/com/github/dockerjava/api/command/ListServicesCmd.java]: the idea is to stop using non collection types

Comment (fengxx) [reply src/main/java/com/github/dockerjava/api/command/ListServicesCmd.java]: IMHO, sometimes varargs is more convenient, e.g. `withIdFilter(id)` is convenient  than cast to List `withIdFilter(Collections.singletonList(id))`, anyway, I removed the varargs

**Pull request merged**